htdocs/include/version.php

```diff
@@ -21,16 +21,16 @@
  * 10 = Final
  */
 
-define('ICMS_VERSION_STATUS', 3);
+define('ICMS_VERSION_STATUS', 2);
 
```

Comment:
2 = beta

Reply:
As we're going for Beta, that's correct.